tests/Fixer/LanguageConstruct/FunctionToConstantFixerTest.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\LanguageConstruct;
+
+use PhpCsFixer\Test\AbstractFixerTestCase;
+
+/**
+ * @author SpacePossum
+ *
+ * @internal
+ */
+final class FunctionToConstantFixerTest extends AbstractFixerTestCase
+{
+    /**
+     * @param string      $expected
+     * @param null|string $input
+     *
+     * @dataProvider provideTestCases
+     */
+    public function testFix($expected, $input = null)
+    {
+        $this->doTest($expected, $input);
+    }
+
+    public function provideTestCases()
+    {
+        return array(
+            'Minimal case, alternative casing, alternative statement end.' => array(
+                '<?php echo PHP_VERSION?>',
+                '<?php echo PHPversion()?>',
+            ),
+            'With embedded comment.' => array(
+                '<?php echo PHP_VERSION/**/?>',
+                '<?php echo phpversion(/**/)?>',
+            ),
+            'With white space.' => array(
+                '<?php echo PHP_VERSION      ;',
+                '<?php echo phpversion  (  )  ;',
+            ),
+            'With multi line whitespace.' => array(
+                '<?php echo
+                PHP_VERSION
+                '.'
+                '.'
+                ;',
+                '<?php echo
+                phpversion
+                (
+                )
+                ;',
+            ),
+            'Global namespaced.' => array(
+                '<?php echo \PHP_VERSION;',
```

Comment:
same as `phpversion()` was translated into `PHP_VERSION`, the `\phpversion()` was updated keeping `\` intact.